### DISCUSSION TITLE
Disable button transition when button is appearing

### DISCUSSION THREAD

**Discussion by yawnak** [saadeghi/daisyui#3208] [Q&A]
I cant find an option to disable button changing size when it is first appearing or loaded in DOM. Help plz!!!

**Comment by saadeghi** [comment]
You can disable the animation completely with `"--animation-btn": "0.25s"`  
https://daisyui.com/docs/themes/#-5

Or with `no-animation` class  
https://daisyui.com/components/button/#button-without-click-animation

But if it has click animation, it will also be visible on page load.  
This behavior will be fixed in daisyUI 5. We can not fix it at this version because the new style will be a breaking change (some sites are depending on customization of that behavior)

Let me know if you have a question.
